app/lib/parseOpeningTimes.js

```diff
@@ -50,7 +50,21 @@ function addPadding(parsedTimes) {
   return parsedTimes;
 }
 
+function isOpen(times) {
+  let openDays = 0;
+  daysOfWeek.forEach((day) => {
```

Comment:
`forEach` could be replaced with `includes` to prevent the full set being iterated.